modules/mod_acl_user_groups/support/acl_user_groups_checks.erl

```diff
@@ -500,7 +506,10 @@ can_rsc(Id, view, Context) when is_integer(Id) ->
     UGs = user_groups(Context),
     (
         can_rsc_1(Id, view, CGId, CatId, UGs, Context)
-        andalso m_rsc:p_no_acl(Id, is_published_date, Context)
+        andalso (
+            is_collab_group_member(CGId, Context)
+            orelse m_rsc:p_no_acl(Id, is_published_date, Context)
+        )
```

Comment:
But for non-collaboration group setups, wouldn’t this deny view permission on unpublished resources? What if you explicitly want members of (e.g.) the managers group to view unpublished resources?